jkube-kit/doc/src/main/asciidoc/inc/build/_index.adoc

```diff
@@ -1,23 +1,41 @@
+ifeval::["{plugin-type}" == "maven"]
 [[build-configuration]]
 === Build Configuration
+endif::[]
+ifeval::["{plugin-type}" == "gradle"]
+[[build-configuration]]
+==== Build Configuration
+endif::[]
+
 
 include::_overview.adoc[]
 
 include::_configuration.adoc[]
 
 [[build-assembly]]
+ifeval::["{plugin-type}" == "maven"]
 === Assembly
+endif::[]
+ifeval::["{plugin-type}" == "gradle"]
+==== Assembly
```

Comment:
Indentation in these headers doesn't seem to be OK:

![image](https://github.com/eclipse/jkube/assets/488391/77d5b914-7259-4371-b33f-9c61813d11c0)

Reply:
These Image Configuration sub-sections seem to be duplicated between `k8sBuild` and this section.

It's better to remove these as they are already moved to `k8sBuild` documentation section.

Reply:
I'm not sure what you mean.
From what I see, the current problem is that they should be declared as h3 and they are h4

Reply:
these sections are  defined in [`k8s:build` section of Kubernetes Maven Plugin ](https://eclipse.dev/jkube/docs/kubernetes-maven-plugin/#jkube:build)
![Screenshot_20240208_190113](https://github.com/eclipse/jkube/assets/13834498/78276ab7-4a6d-4349-8350-768be9612527)

After moving documentation of both `k8s:build` and `k8sBuild` to `jkube-kit/doc` , now image configuration sections are duplicated in both `k8sBuild` section and Gradle Groovy DSL configuration.

Reply:
You're right, they should be h3 if they are in [Gradle Groovy DSL configuration](https://eclipse.dev/jkube/docs/kubernetes-gradle-plugin/#groovy-configuration) section. 

It would be better if these configuration option are defined at same place for both plugins. 

Do we want to put these inside `k8s:build` / `k8sBuild` or in a separate [Global Configuration options section](https://eclipse.dev/jkube/docs/kubernetes-gradle-plugin/#groovy-configuration) like we have in Kubernetes Gradle Plugin?

Reply:
We can't do this now
I had the same issue with Helm (see `// TODO` comments I had to add to deal with this)
We can (must) do a more in depth documentation refactor in the future.

Reply:
I've reverted Gradle Groovy DSL configuration section to what it was before:

![Screenshot_20240208_225233](https://github.com/eclipse/jkube/assets/13834498/c374f82c-57f4-40d1-a49c-c40aec0f8420)